shared-signals/README.md

```diff
@@ -0,0 +1,13 @@
+# About
+
+Sets up common infrastructure used for implemneting Shared Signals
+
+# Deploying
+
+The CI/CD pipeline will deploy the resources in the dev environment as well as all the other environments from build to production. To manually deploy in the dev environment, you can use the following command:
+
+```bash
+sam deploy --stack-name shared-signals-infra --parameter-overrides ParameterKey=Environment,ParameterValue=dev --resolve-s3 --capabilities CAPABILITY_NAMED_IAM --template-file "shared-signals/template.yaml"
```

Comment:
I'm not sure we want to encourage people to do this, because it'll potentially lead to people making changes in branches that are temporary and overwriting the "official" version.

Reply:
i needed to deploy manually while playing around with dynamo.  i can add a warning to use a different stack name